tuf/api/metadata.py

```diff
@@ -120,18 +121,14 @@ def __init__(self, signed: T, signatures: Dict[str, Signature]):
     def from_dict(cls, metadata: Dict[str, Any]) -> "Metadata[T]":
         """Creates Metadata object from its dict representation.
 
-        Arguments:
+        Args:
             metadata: TUF metadata in dict representation.
 
-        Raises:
-            KeyError: The metadata dict format is invalid.
-            ValueError: The metadata has an unrecognized signed._type field.
-
```

Comment:
@jku I think we agreed we want to document `KeyError` and `ValueError` from `from_dict()` and the constructors?
That's why I created #1819.

Reply:
@ivanayov @jku has agreed we want to document those exceptions.
I will fix that when proposing a pr for #1819.
Could you remove that change?

Reply:
@MVrachev yep, I revert then.